### PR TITLE
 Issue #17 Recently Verified

### PR DESCRIPTION
**Pull request by codelear**
added checkbox to filter data verified in the last 2 hours

image

### CODE REVIEW

**Comment by codelear** [comment]
@hargup I created 3 PRs for open issues. #17 , #19, #26

**Comment by hargup** [comment]
Thank you @codelear will check them out :)

File: web/src/components/MobileCovid.js [hunk 1]
```diff
@@ -102,6 +104,10 @@ export default function MobileCovid() {
     .slice()
     .sort((a, b) => (b.verifiedAt || "").localeCompare(a.verifiedAt || ""));
 
+   const dataVerifiedLastTwoHours = displayData.slice().filter(data=>{
+    return ((data.verifiedAt||addToCurrentTime(-3,0).toISOString()) >= addToCurrentTime(-2,0).toISOString())
```
Comment (hargup) [inline web/src/components/MobileCovid.js:108]: 👍

**Comment by hargup** [comment]
LGTM

**Pull request merged**